testing/examples/run_instruction_arbitrary/cloudbuild.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Tests that verify the example manipulating arbitrary commands using
+# rules_docker and Dockerfile.
+
+timeout: 21600s
```

Comment:
same q as above

Reply:
Timeout is reduced.
High CPU here brings the test time from almost 20 mins to just under 10 mins.